db/migrate/20201005181510_change_display_when_signed_in_column_name.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeDisplayWhenSignedInColumnName < ActiveRecord::Migration[6.0]
+  def change
+    safety_assured { rename_column :navigation_links, :display_when_signed_in, :display_only_when_signed_in }
```

Comment:
I felt like this column name didn't make any sense and so I've renamed it. Since this is a brand new feature we dont have datd for it as yet. I haven't created a frontend as yet.